src/components/templates/AvatarGrid/RoomModal.tsx

```diff
@@ -48,16 +48,18 @@ export const RoomModal: React.FC<PropsType> = ({
   const dispatch = useDispatch();
 
   const { user, profile } = useUser();
-  const partygoers = usePartygoers();
+  const { recentWorldUsers } = useRecentWorldUsers();
   const venueEvents = useSelector(venueEventsSelector) ?? [];
   const venue = useSelector(venueSelector);
   const venueName = venue?.name;
   const roomTitle = room?.title;
 
   const usersInRoom = useMemo(
     () =>
-      partygoers.filter((goer) => goer.room === `${venueName}/${roomTitle}`),
-    [partygoers, roomTitle, venueName]
+      recentWorldUsers.filter(
+        (user) => user.room === `${venueName}/${roomTitle}`
+      ),
+    [recentWorldUsers, roomTitle, venueName]
   );
```

Comment:
This would be resolved by a small hook such as `recentUsersInRoom(venueId, roomName)`, which would probably consume `recentUsersInVenue(venueId)`

Reply:
https://github.com/sparkletown/sparkle/pull/1047#discussion_r561862284